django/db/models/base.py

```diff
@@ -204,7 +204,7 @@ def __new__(cls, name, bases, attrs):
             if base not in parents or not hasattr(base, '_meta'):
                 # Things without _meta aren't functional models, so they're
                 # uninteresting parents.
-                inherited_attributes |= set(base.__dict__.keys())
+                inherited_attributes.update(base.__dict__.keys())
```

Comment:
You can remove the `keys()` call as `iter(dict.keys())` yields the same results as `iter(dict)`. That is `update(base.__dict__)`.